src/main/java/com/invicto/streamingPlatform/web/config/WebSecurityConfig.java

```diff
@@ -13,26 +17,34 @@
     private CustomAuthenticationProvider customAuthenticationProvider;
 
     @Override
-    public void configure(AuthenticationManagerBuilder auth) {
+    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
         auth.authenticationProvider(customAuthenticationProvider);
     }
 
     @Override
     protected void configure(HttpSecurity httpSecurity) throws Exception {
         httpSecurity
+                .csrf().disable()
                 .authorizeRequests()
-                    .antMatchers("/signup", "/", "/styles/**", "/images/**").permitAll()
+                    .antMatchers("/signup", "/forgot_password", "/reset_password", "/", "/styles/**", "/images/**").permitAll()
```

Comment:
Перевір чи клієнт отримує js файл. Мені здається, що потрібно додати цей шлях до виключень:
```suggestion
                    .antMatchers("/signup", "/forgot_password", "/reset_password", "/", "/styles/**", "/js/**", "/images/**").permitAll()
```